cves/kernel/CVE-2020-11494.yml

```diff
@@ -110,17 +106,17 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: a1044e36e457fb6dbdf90ce756d578b251d99b5e
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually Confirmed
 - commit: 87397fe10db0e7ee85041eee5a40052cab66aaff
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually Confirmed
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 
```

Comment:
upvote + 2

Reply:
+2 upvote. Not super interesting to me honestly